index.mjs

```diff
@@ -3,6 +3,7 @@ import koaSession from 'koa-session';
 import nanoid from 'nanoid';
 import fetch from 'node-fetch';
 import fs from 'fs';
+import { URLSearchParams } from 'url';
```

Comment:
Usage of old url module is not encouraged by node anymore. Use globally available WHATWG api instead (https://nodejs.org/api/url.html#url_url_strings_and_url_objects)